src/Symfony/Component/Workflow/CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ CHANGELOG
 
  * Added context to `TransitionException` and its child classes whenever they are thrown in `Workflow::apply()`
  * Added `Registry::has()` to check if a workflow exists
- * Added support for `$context[Workflow::DISABLE_ANNOUNCE_EVENT] = true` when calling `workflow->apply()` to not fire the announce event
+ * Added support for specifying which events should be dispatched when calling `workflow->apply()`
```

Comment:
💎

Reply:
Indeed, this is a BC Break.

IHMO, you should merge the default configuration (the one passed to the construct) with the one in the `$context`